v3/tasks/20250502-v3-fee-percentages-provider/input.ts

```diff
@@ -0,0 +1,17 @@
+import { Task, TaskMode } from '@src';
+
+export type ProtocolFeePercentagesProviderDeployment = {
+  Vault: string;
+  ProtocolFeeController: string;
+  BalancerContractRegistry: string;
+};
+
+const Vault = new Task('20241204-v3-vault', TaskMode.READ_ONLY);
+const ProtocolFeeController = new Task('20250214-v3-protocol-fee-controller-v2', TaskMode.READ_ONLY);
```

Comment:
I'd get the protocol fee controller from the vault directly just to prevent version issues with this.